lib/doc2pdf.rb

```diff
@@ -21,7 +21,9 @@ def self.extract(document:)
   def self.replace!(document:, replacer:)
     DocumentTraversal.new(document: document).each do |item|
       search(item.text).each do |occurrence|
-        item.substitute(occurrence, replacer.call(key: occurrence))
+        tm = replacer.call(key: occurrence)
+        puts "Occurrence #{occurrence}, Replace #{tm}"
```

Comment:
Please remove debugging code.
Also, regarding `tm`, you could use a more descriptive name.

Reply:
As before, wrong commit